components/Ripple/examples/StatefulRippleExample.swift

```diff
@@ -49,7 +49,13 @@ class RippleView : UIView {
     statefulRippleView.isRippleHighlighted = true
   }
 
+  override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
+    statefulRippleView.superviewTouchMoved(touches.first, event: event)
```

Comment:
Consider replacing superview with parentView to avoid subclass confusion.

Naming is hard!

Reply:
I named it superview as that is the way UIKit calls view's parents. parentView is more of a view controller notion. Do you feel strongly this should be parent?

Reply:
Blink, blink.

Ignore that, need more caffeine!